### PR TITLE
Creates the Alibaba OSS Target Documentation

### PR DESCRIPTION
**Pull request by JeffNeff**
Relates to:
https://github.com/triggermesh/knative-targets/pull/459
https://github.com/triggermesh/frontend/pull/2034

### CODE REVIEW

**Comment by JeffNeff** [comment]
Where are the secret icons located?

File: docs/targets/alibabaoss.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+# Event Target for Alibaba OSS 
+
+This event target receives [CloudEvents][ce] over HTTP and sends them to [Alibaba OSS](https://www.alibabacloud.com/product/oss)
```
Comment (cab105) [inline docs/targets/alibabaoss.md:3]: How about following the same pattern as the `CloudEvents` reference by moving the Alibaba OSS URL to the bottom of the doc?

**Review by cab105 — APPROVED** [review]
Left one suggestion, but it could apply to the other Alibaba references.

File: docs/targets/alibabaoss.md (hunk 1 above)
Comment (JeffNeff) [reply docs/targets/alibabaoss.md]: done thank you

**Comment by JeffNeff** [comment]
dam i missed that comment.. i only saw the one.. I'm sorry

**Pull request merged**